### PR TITLE
Update where probe-scraper looks for sync metrics

### PR DESCRIPTION
**Pull request by skhamis**
The sync team is trying to consolidate metrics for both iOS and Android and thus requiring to move the metrics.yaml to application-services repo so that they can share the same file. This updates where the probe-scraper looks for the `metrics.yaml`

app-services PR (merged): https://github.com/mozilla/application-services/pull/5624
firefox-android PR: https://github.com/mozilla-mobile/firefox-android/pull/2117

### CODE REVIEW

**Comment by Dexterp37** [comment]
@badboy isn't this related to some bug you "recently" looked at?

**Comment by badboy** [comment]
Yup: https://bugzilla.mozilla.org/show_bug.cgi?id=1827949
Short of renaming the metric the metrics that clash we cannot support it.

File: repositories.yaml
Comment (relud) [inline repositories.yaml:68]: CI is failing because this name also needs to be changed on fenix and firefox_reality

**Review by relud — DISMISSED** [review]
r+wc

**Comment by skhamis** [comment]
No worries! This won't land until after the main two land, appreciate the reviews! I wanted to ensure I get this up early to ensure I can course correct easier than fumbling around -- 

> Yup: https://bugzilla.mozilla.org/show_bug.cgi?id=1827949 Short of renaming the metric the metrics that clash we cannot support it.

yes indeed, once we make this move we'll rename the metrics with the `_v2` guidance and it'll be easier for us since we'll have both android and iOS sharing the same metrics.yaml so the renaming only needs to happen once

**Comment by skhamis** [comment]
@badboy now that https://github.com/mozilla/application-services/pull/5624 landed, I imagine this needs to land before: https://github.com/mozilla-mobile/firefox-android/pull/2117 given that it's failing on that side due to the removal of the `sync-telemetry` module. Though I wonder if it's better to leave the sync-telemetry module in android for now and just switch the calls to app-services? Curious the best way from a glean perspective.

**Comment by badboy** [comment]
> @badboy now that [mozilla/application-services#5624](https://github.com/mozilla/application-services/pull/5624) landed, I imagine this needs to land before: [mozilla-mobile/firefox-android#2117](https://github.com/mozilla-mobile/firefox-android/pull/2117) given that it's failing on that side due to the removal of the `sync-telemetry` module. Though I wonder if it's better to leave the sync-telemetry module in android for now and just switch the calls to app-services? Curious the best way from a glean perspective.
> 
> Edit: and remove the sync-telemetry module in a separate patch after everything lands.

A dry-run locally with this patch works. 
Switching over and removing the files in one go is fine, the data will also be fine, we never "forget" old information, so the system will continue to know about the old metrics just fine.

Is it ok if we land this only on Monday? While I'm nearly confident this all works out, I'd rather not have that merge before the weekend just in case.

**Comment by badboy** [comment]
> > > @badboy now that [mozilla/application-services#5624](https://github.com/mozilla/application-services/pull/5624) landed, I imagine this needs to land before: [mozilla-mobile/firefox-android#2117](https://github.com/mozilla-mobile/firefox-android/pull/2117) given that it's failing on that side due to the removal of the `sync-telemetry` module. Though I wonder if it's better to leave the sync-telemetry module in android for now and just switch the calls to app-services? Curious the best way from a glean perspective.
> > > Edit: and remove the sync-telemetry module in a separate patch after everything lands.
> > 
> > 
> > A dry-run locally with this patch works. Switching over and removing the files in one go is fine, the data will also be fine, we never "forget" old information, so the system will continue to know about the old metrics just fine.
> > Is it ok if we land this only on Monday? While I'm nearly confident this all works out, I'd rather not have that merge before the weekend just in case.
> 
> No problem! Happy to land whenever the team is comfortable, [mozilla-mobile/firefox-android#2117](https://github.com/mozilla-mobile/firefox-android/pull/2117) got approved on that side -- so it's all ready to start landing!

Thanks, this got lost in the past 2 days because of other fires. Landing it now!

**Review by badboy — APPROVED** [review]

**Pull request merged**
